### PR TITLE
docs : Added documentation to Notification package

### LINKED ISSUE #185: Document the files in mifopay/notification package
**Summary:**

Document the files in mifopay/notification package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/notification

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes #185 
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@luckyman20 Please review this PR.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/notification/NotificationContract.java [hunk 1]
```diff
@@ -7,16 +7,25 @@
 import java.util.List;
 
 /**
- * Created by ankur on 24/July/2018
+ * This is a contract class working as an Interface for UI
+ * and Presenter components of the Notification Architecture.
```
Comment (codepoet2017390) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/notification/NotificationContract.java]:
```suggestion
 * and Presenter components of the Notification package.
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/notification/ui/NotificationActivity.java [hunk 2]
```diff
@@ -57,13 +59,20 @@ protected void onCreate(Bundle savedInstanceState) {
         mNotificationPresenter.fetchNotifications();
     }
 
+    /**
+     * A function to setup the Layout Manager and integrate the RecyclerView with Adapter.
+     * This function also add divider between notifications vertically
```
[the remark this answers is not in the document]
Comment (RajaVamsi11) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/notification/ui/NotificationActivity.java]: done

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/notification/NotificationContract.java (hunk 1 above)
Comment (RajaVamsi11) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/notification/NotificationContract.java]: done

**Comment by RajaVamsi11** [comment]
@codepoet2017390 Updated according to the changes.

**Comment by rachittt** [comment]
Hey @codepoet2017390 , there has been a lot of changes in the project and thus we need a fresh start for the documentation part. 😄

**Pull request closed without merging**